model-optimizer/extensions/front/onnx/clip_ext.py

```diff
@@ -33,5 +33,8 @@ def extract(cls, node):
             }
             AttributedClamp.update_node_stat(node, attrs)
         else:
+            assert not onnx_node_has_attr(node, 'min') and not onnx_node_has_attr(node, 'max'), \
```

Comment:
Because from the MO side we handle situation correctly (we use inputs instead of attributes "min" and "max") and the error will be displayed to the user so user will be notified.